sys-apps/systemd/systemd-9999.ebuild

```diff
@@ -173,6 +180,13 @@ fi
 	# Bug 463376
 	sed -i -e 's/GROUP="dialout"/GROUP="uucp"/' rules/*.rules || die
 
+	# missing in tarball
+	cp "${FILESDIR}"/217-systemd-consoled.service.in \
+		units/user/systemd-consoled.service.in || die
+
+	# https://github.com/coreos/bugs/issues/220
+	epatch "${FILESDIR}"/218-0001-networkd-accept-a-trailing-.-on-the-end-of-domains.patch
```

Comment:
A different fix went in so this patch can be dropped.

https://github.com/systemd/systemd/commit/f50f01f4b738f2f00b30d0e02e8cf54ab99a9f27